Makefile

```diff
@@ -120,9 +120,12 @@ test: generate fmt vet manifests
 
 test-with-vendor: generate fmt vet manifests
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh.
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test -mod=vendor ./... -coverprofile cover.out; go tool cover -html=cover.out -o coverage.html
 
+#	go test -mod=vendor ./... -coverprofile cover.out;
+#	go tool cover -html=cover.out -o coverage.html
+
```

Comment:
Uncomment if 'setup-envtest' is not used & when controller control plane error is fixed

Reply:
https://github.com/Orange-OpenSource/nifikop/blob/2833a4941b55dd36509884b6d652337abe54f0fc/Makefile#L124

Here, first 3 commands are in error ('source not found,and so on) + in 1 line only the third command exit code is used to determine if the test passed on CircleCI (so it's pretty much always green except sys error)